frontend/packages/dev-console/integration-tests/README.md

```diff
@@ -1,31 +1,78 @@
 # Getting Started
 
+- [Cypress Cucumber Hand book](https://docs.google.com/document/d/1hL_k5r6CVxbY5va6RPPCJfndQjLwSdt6SUKj-kWLqig/edit#)
+
+## Setup
+
+### Include Extensions
+
+"alexkrechik.cucumberautocomplete",
+"dylanmoerland.cypress-cucumber-steps"
+"streetsidesoftware.code-spell-checker"
```

Comment:
Render this as list? Currently this looks like this:

![image](https://user-images.githubusercontent.com/139310/106261278-a5cc3b00-6221-11eb-819d-021428770dce.png)

```suggestion
* "alexkrechik.cucumberautocomplete",
* "dylanmoerland.cypress-cucumber-steps"
* "streetsidesoftware.code-spell-checker"
```